### DISCUSSION TITLE
Question: could we add custom label to polygon ?

### DISCUSSION THREAD

**Discussion by hengseyha** [inocan-group/vue3-google-map#212] [Q&A]

**Comment by HusamElbashir** [comment]
Can you please provide more info so people could help you?

**Comment by mad-hengseyha** [comment]
<img width="467" alt="Screenshot 2023-06-10 at 1 10 38 PM" src="https://github.com/inocan-group/vue3-google-map/assets/108649284/e90561be-bd3e-455a-ba9b-b19d2ef797f6">

like this photo
